databricks_cli/execution_context/cli.py

```diff
@@ -0,0 +1,251 @@
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import click
+from tabulate import tabulate
+from databricks_cli.click_types import (
+    ClusterIdClickType, CommandIdClickType, 
+    CommandOutputType, CommandStringType,
+    ExecutionContextIdClickType, OutputClickType
+)
+
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS, pretty_format, truncate_string
+from databricks_cli.version import print_version_callback, version
+from databricks_cli.configure.config import provide_api_client, profile_option, debug_option
+from databricks_cli.execution_context.api import ExecutionContext, ExecutionContextApi
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help="Creates a new execution context.")
+@click.option('--cluster-id', required=True, type=ClusterIdClickType(),
+              help=ClusterIdClickType.help)
+@click.option('--language', required=False, default="python", 
+              type=click.Choice(['python', 'scala', 'sql'], case_sensitive=False),
+              help="The language for the context.")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_context_cli(api_client, cluster_id, language): #  NOQA
+    """
+    Creates a new execution context.
+    """
+    result = ExecutionContextApi(api_client).create_context(cluster_id, language)
+    click.echo(pretty_format(result))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help="Gets information about an execution context.")
+@click.option('--cluster-id', required=True, type=ClusterIdClickType(),
+              help=ClusterIdClickType.help)
+@click.option('--context-id', required=True, type=ExecutionContextIdClickType(),
+              help=ExecutionContextIdClickType.help)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_context_status_cli(api_client, cluster_id, context_id): #  NOQA
```

Comment:
this command makes no sense in isolation within the CLI from end-user perspective.